interfaces/acados_template/acados_template/acados_model.py

```diff
@@ -71,6 +71,14 @@ def __init__(self):
         The time dependency can be used within cost formulations and is relevant when cost integration is used.
         Start times of shooting intervals can be added using parameters.
         """
+        self.p_global = None
+        """
+        CasADi variable containing parameters that change rarely;
```

Comment:
maybe first mention that these parameters are global? it might also be computationally beneficial to use them if they change often as they don't have to be set for each node individually

Reply:
and aren't the computations performed in `set_p_global`? instead of custom update?

Reply:
Oh yes, sorry, this was actually already in a commit that I forgot to push. Thanks!